### PR TITLE
[EPMRPP-49163][Cucumber] Launch logs example

### PR DESCRIPTION
**Pull request by ihar-kahadouski**

### CODE REVIEW

File: example-cucumber/src/test/java/com/epam/reportportal/example/cucumber/logging/LaunchLog.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+/**
+ * @author <a href="mailto:ihar_kahadouski@epam.com">Ihar Kahadouski</a>
+ */
+public class LaunchLog {
+
+	private static final Logger LOGGER = LoggerFactory.getLogger(LaunchLog.class);
+
+	@Given("I attach files to launch logs")
+	public void reportStep() {
+		Arrays.stream(Attachment.values()).forEach(it -> launchLogWithAttachment(randomLogLevel(), FILE_NAME, it));
```
Comment (HardNorth) [inline example-cucumber/src/test/java/com/epam/reportportal/example/cucumber/logging/LaunchLog.java]: Let it be more simple and understandable for everyone. Please do not use streams here, just a plain java and would be enough.

File: example-cucumber/src/test/java/com/epam/reportportal/example/cucumber/logging/LaunchLog.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+		LOGGER.info("Add logs with attachments to launch");
```
Comment (HardNorth) [inline example-cucumber/src/test/java/com/epam/reportportal/example/cucumber/logging/LaunchLog.java:25]: I think this line should be before attachment logging.

**Review by HardNorth — CHANGES_REQUESTED** [review]

File: example-cucumber/src/test/java/com/epam/reportportal/example/cucumber/logging/LaunchLog.java (hunk 1 above)
Comment (ihar-kahadouski) [reply example-cucumber/src/test/java/com/epam/reportportal/example/cucumber/logging/LaunchLog.java]: done

File: example-cucumber/src/test/java/com/epam/reportportal/example/cucumber/logging/LaunchLog.java (hunk 2 above)
Comment (ihar-kahadouski) [reply example-cucumber/src/test/java/com/epam/reportportal/example/cucumber/logging/LaunchLog.java:25]: fixed

**Review by HardNorth — APPROVED** [review]

**Pull request merged**